class BaseError(Exception):
    pass


class KeyExistsError(BaseError):
    pass
